lib/Local.js

```diff
@@ -179,30 +184,27 @@ function Local(){
         break;
 
       case 'forceproxy':
+      case 'forceProxy':
         if(value)
-          this.forceProxyFlag = '-forceproxy';
-        break;
-
-      case 'hosts':
-        if(value)
-          this.hosts = value;
+          this.forceProxyFlag = '--force-proxy';
         break;
 
       case 'logfile':
+      case 'logFile':
         if(value)
           this.logfile = value;
         break;
 
-      case 'binarypath':
```

Comment:
@Blackbaud-PatrickOFriel `binarypath` is an option provided by this wrapper not the actual binary. Can you please add it back in the PR?